youtube_dl/YoutubeDL.py

```
@@ -758,6 +758,15 @@ def extract_info(self, url, download=True, ie_key=None, extra_info={},
                                     'and will probably not work.')
 
             try:
+                if self.original_ie:
+                    pass # self.original_ie already exists
+            except AttributeError:
+                try:
+                    self.original_ie = re.search("\.([^\.]+)'>", str(type(ie))).group(1)
+                except:
```

Never ever use bare except.